target/linux/ramips/dts/mt7628an_kroks_kndrt31r19.dts

```diff
@@ -0,0 +1,75 @@
+#include "mt7628an_kroks_1eth.dtsi"
+
+/ {
+	compatible = "kroks,kndrt31r19", "mediatek,mt7628an-soc";
+	model = "Kroks KNdRt31R19";
```

Comment:
This is completely different from the title in the PR?

Reply:
Well, it seems like the device is actually has 2 names - the internal one here and some marketing name in the title.
Both names are used in original commit: 
`DEVICE_TITLE := Kroks Rt-Pot mXw DS RSIM`:
https://github.com/kroks-free/openwrt/commit/ba19321f71c680cbbec2da5d60ee5f9947a33eca#diff-bb176229946c2cb6b2c4899e91408e84b49fe2518003b294e577f8605145c220R187
and
`model = "Kroks KNdRt31R19";` here:
https://github.com/kroks-free/openwrt/commit/ba19321f71c680cbbec2da5d60ee5f9947a33eca#diff-c30ff05179792fdffab9c2bdef800c31b9ad56ddf9366da24f29e3af86d4d045R5

I don't know how to handle that properly, but I think keeping them both might help to identify the device.